priam/src/main/java/com/netflix/priam/identity/InstanceIdentity.java

```diff
@@ -104,7 +107,7 @@ public InstanceIdentity(IPriamInstanceFactory factory, IMembership membership, I
         init();
     }
 
-    public PriamInstance getInstance() {
+    PriamInstance getInstance() {
```

Comment:
Token was previously used both as the initial_token and as the identifier of a node's backup in the context of the entire cluster. 

When using vnodes, I don't need an initial token, but I do still need an identifier for the backup, so I wanted to separate these two concerns. Because the backup identifier is redundant with data already in SDB, it felt better not to put in in the PriamInstance class, and instead use PriamInstance purely as the Data Access DTO, but InstanceIdentity as the business model.